fuse/common.py

```diff
@@ -24,14 +24,12 @@ def dynamic_chunking(data, scale, n_min):
         if value <= scale:
             clusters.append(c)
             n_cluster += 1
-
         elif n_cluster + 1 < n_min:
             clusters.append(c)
             n_cluster += 1
-
         elif value > scale:
-            c = c + 1
             clusters.append(c)
+            c = c + 1
             n_cluster = 0
```

Comment:
Why was this line moved? I think we want to start a new cluster if `value > scale` and for that reason we need to bump the cluster number `c` before appending it or?

Reply:
I am wrong.